__tests__/app.test.js

```diff
@@ -335,3 +335,44 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+describe("DELETE /api/comments/:comment_id", () => {
+  test.only("returns an object", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(200)
+      .then(({ body }) => {
+        const comment = body.comment;
+        expect(typeof comment).toBe("object");
```

Comment:
We don't really need this test. The test below is showing that `comment` is an object, as we're using dot notation on it in the tests. 👍